coordinatorlib/src/main/scala/com/socrata/datacoordinator/truth/metadata/CopyInfo.scala

```diff
@@ -25,22 +24,6 @@ case class UnanchoredCopyInfo(@JsonKey("sid") systemId: CopyId,
 object UnanchoredCopyInfo extends ((CopyId, Long, LifecycleStage, Long, DateTime) => UnanchoredCopyInfo) {
   override def toString = "UnanchoredCopyInfo"
 
-  private implicit object DateTimeCodec extends JsonDecode[DateTime] with JsonEncode[DateTime] {
```

Comment:
these are not actually identical: https://github.com/socrata-platform/socrata-thirdparty-utils/blob/master/core/src/main/scala/com/socrata/thirdparty/json/AdditionalJsonCodecs.scala#L8

I'm gonna ponder the differences a little.